app/controllers/reputation_controller.rb

```diff
@@ -16,7 +20,11 @@ def reply
     if user_signed_in?
       @reply = Reply.find(params[:id])
       @evaluation_value = params[:up_down] == 'down' ? 0 : 1
-      @reply.add_or_update_evaluation(:likes, @evaluation_value, current_user)
+      if params[:up_down] == 'down'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.